SimTransport/PPSProtonTransport/interface/ProtonTransport.h

```diff
@@ -0,0 +1,62 @@
+#ifndef PROTONTRANSPORT
+#define PROTONTRANSPORT
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "HepMC/GenEvent.h"
+#include "CLHEP/Random/RandGauss.h"
+#include <CLHEP/Vector/LorentzVector.h>
+#include "CLHEP/Units/GlobalSystemOfUnits.h"
```

Comment:
@mundim , these CLHEP headers should not be used in *.h files, if they really needed inside .h one has to use <CLHEP/Units/GlobalSystemOfUnits.h>. Ideally they should go into *.cc